docs/3.0/deploy/index.mdx

```diff
@@ -230,6 +230,12 @@ class Deployment:
     paused: bool = False
     trigger: Trigger = None
 
+    # concurrency limiting
+    concurrency_limit: Optional[int] = None
+    concurrency_options: Optional[
+        ConcurrencyOptions(collision_strategy=Literal['ENQUEUE', 'CANCEL_NEW'])
+    ] = None
+
```

Comment:
![image](https://github.com/user-attachments/assets/e901bda0-bff0-4a4c-8d79-7551b7e439a1)

The strategy value is actually supposed to take an Enum, but using literals here for brevity (and they should technically still work)